### PR TITLE
[Ready] 2020 campinas initial site content 

### PR DESCRIPTION
**Pull request by pasimoes**
Please create Site for KCD 2020 Campinas

### CODE REVIEW

**Comment by pasimoes** [comment]
Issues Fixed

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@ yarn-error.log
 
 # Ignore Dropbox files
 .DS_Store
+content/events/2020-amsterdam/.speakers.md.swp
```
Comment (lucperkins) [inline .gitignore]: Please delete this file rather than adding it to the gitignore

Comment (pasimoes) [reply .gitignore]: Done. Thank you

File: content/events/2020-campinas/_index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+social:
+  twitter: https://twitter.com/kcdcampinas
+sponsors:
+- level: Initial sponsors
+  orgs:
+  - name: Mandic
+    logo: sponsors/mandic.svg
+    url: https://www.mandic.com.br/
+  - name: PagSeguro
+    logo: sponsors/pagseguro.png
```
Comment (KTan1226) [inline content/events/2020-campinas/_index.md]: Please change this file to SVG.

File: content/events/2020-campinas/_index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    url: https://www.pagseguro.com.br
+  - name: Aqua
+    logo: sponsors/aquasec.png
```
Comment (KTan1226) [inline content/events/2020-campinas/_index.md]: Please also change this file to SVG.

File: content/events/2020-campinas/_index.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  twitter: https://twitter.com/gabyldias
+  homepage: https://github.com/floord
+- name: Carol Vilas Boas
+  image: team/carol.jpeg
+  twitter: https://twitter.com/cavillasboas
+  homepage: https://www.linkedin.com/in/carolfvb/
+organizers_email: organizers-campinas@kubernetescommunitydays.org
+local_law_enforcement: 190
+medical_emergency: 193
+cfp_deadline: '2020-03-13'
```
Comment (KTan1226) [inline content/events/2020-campinas/_index.md]: Please add the link to your CFP: https://kcdcfpsubmissions.smapply.io/prog/kcd_campinas/. I am reaching out with instructions on admin access. Thanks!

Comment (pasimoes) [reply content/events/2020-campinas/_index.md]: Kristi as we started before CNCF setting up all tools, we already opened CFP on papercall.io .

We are very close to end date. We really should change?

https://www.papercall.io/kcd-campinas

Comment (KTan1226) [reply content/events/2020-campinas/_index.md]: No problem- feel free to use Papercall given the short timeline. Thank you for your understanding.

**Comment by KTan1226** [comment]
@pasimoes Can you please add an LGTM from one of your other organizers? We can merge once confirmed. Thanks!

File: content/events/2020-campinas/_index.md (hunk 2 above)
Comment (pasimoes) [reply content/events/2020-campinas/_index.md]: Done! Please review.

File: content/events/2020-campinas/_index.md (hunk 3 above)
Comment (pasimoes) [reply content/events/2020-campinas/_index.md]: done.

**Comment by eduardo-eth0** [comment]
LGTM =)

**Pull request merged**